presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMultimapAggAggregation.java

```diff
@@ -130,4 +205,36 @@ public void testDoubleRowMap()
 
         assertAggregation(aggFunc, map.isEmpty() ? null : map, builder.build());
     }
+
+    private static <K, V> void testMultimapAggWithGroupBy(
```

Comment:
This function did two things:
1. Convert `expectedKeys` and `expectedValues` to `expectedMultimap`
2. Assert the aggregation execution does return `expectedMultimap`.

I suggest to have the standalone function `createMultimapFromKeyValuePairs`, and do the assert in each test.

Reply:
Note this doesn't follow the convention in the rest of the test which uses a `testMultimapAgg` that both constructs input and asserts output